app/decorators/miq_template_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class MiqTemplateDecorator < MiqDecorator
+class MiqTemplateDecorator < VmOrTemplateDecorator
```

Comment:
I'm afraid this will break the functionality of decorators, will have to test it.